gravity-forms-multi-currency.php

```diff
@@ -63,11 +63,11 @@ public function form_process()
     {
         $form_id = isset($_POST["gform_submit"]) ? $_POST["gform_submit"] : 0;
         if ($form_id) {
-            $form_info = RGFormsModel::get_form($form_id);
+            $form_info = GFAPI::get_form($form_id);
             $is_valid_form = $form_info && $form_info->is_active;
 
             if ($is_valid_form) {
-                $form = RGFormsModel::get_form_meta($form_id);
+                $form = GFAPI::get_form($form_id);
```

Comment:
No need to get the form again if we already got in on line 66. Does this get the form meta data as well?

Reply:
It doesn't work anymore. Could you please fix it to work with latest WP & GF versions. Thanks